templates/deployment.yaml

```diff
@@ -233,3 +237,178 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+
+# Netbox Worker
```

Comment:
I think it would be nicer to move the worker into its own template file, if only to avoid this one being so long.

Reply:
done